clients/fossid-webapp/src/test/kotlin/FossIdClientReturnTypeTest.kt

```diff
@@ -196,4 +197,10 @@ class FossIdClientReturnTypeTest : StringSpec({
             }
         }
     }
+
+    "When the scan to delete does not exist, no exception is thrown" {
+        service.deleteScan("", "", SCAN_CODE_1).shouldNotBeNull().run {
+            error shouldBe "Classes.TableRepository.row_not_found"
+        }
```

Comment:
You should not need run:
```kotlin
service.deleteScan("", "", SCAN_CODE_1) shouldNotBeNull {
  error shouldBe "Classes.TableRepository.row_not_found"
}
```

Reply:
If you look at @sschuberth changes in the same file, this is intentional to prevent creating a dependency.